src/core/annotation.js

```diff
@@ -237,32 +303,6 @@ var Annotation = (function AnnotationClosure() {
       }
     },
 
-    isInvisible: function Annotation_isInvisible() {
-      var data = this.data;
-      return !!(data &&
-                data.annotationFlags &&            // Default: not invisible
-                data.annotationFlags & 0x1);       // Invisible
-    },
-
-    isViewable: function Annotation_isViewable() {
-      var data = this.data;
-      return !!(!this.isInvisible() &&
-                data &&
-                (!data.annotationFlags ||
-                 !(data.annotationFlags & 0x22)) &&  // Hidden or NoView
-                data.rect);                          // rectangle is necessary
-    },
-
-    isPrintable: function Annotation_isPrintable() {
-      var data = this.data;
-      return !!(!this.isInvisible() &&
```

Comment:
Addressed in the new commit. I have restored the previous checks as they are correct according to the spec: an annotation should be printed if the `PRINT` flag is set, if it is not hidden and if it is not invislble. The `NOVIEW` flag indeed has nothing to do with printing, so we do not check that.